frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -264,8 +264,10 @@ export const getMarkerIcon = (feature: ICluster, selected?: boolean) => {
           return landErpIconSelect;
       }
     } else if (
-      selectedClassificationId === Classifications.SurplusActive ||
-      selectedClassificationId === Classifications.SurplusEncumbered
+      (selectedClassificationId === Classifications.SurplusActive ||
+        selectedClassificationId === Classifications.SurplusEncumbered) &&
+      projectWorkflow &&
+      [Workflows.SPL].includes(projectWorkflow)
```

Comment:
After talking with Devin and Bobbi, we determined that we only want the pin to show as purple it has been approved and is in the SPL workflow